pkg/antctl/raw/check/cluster/test_checkcniexistence.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2024 Antrea Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cluster
+
+import (
+	"context"
+	"fmt"
+	"sort"
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"antrea.io/antrea/pkg/antctl/raw/check"
+)
+
+type checkCNIExistence struct{}
+
+func init() {
+	RegisterTest("Check if another CNI is present", &checkCNIExistence{})
+}
+
+func (t *checkCNIExistence) Run(ctx context.Context, testContext *testContext) error {
+	pods, err := testContext.client.CoreV1().Pods(testContext.namespace).List(ctx, metav1.ListOptions{LabelSelector: "name=check-cluster"})
```

Comment:
do you think we should pick a Deployment Pod during setup and use it consistently, rather than duplicate this code in `checkCNIExistence` and `checkOVSLoadable`?

Reply:
yes, could be a `*corev1.Pod` pointer to be consistent with what you did for the other command
or just the Pod name if you prefer